frontend/src/pages/organization/SecretScanningPage/components/ExportResultsModal.tsx

```diff
@@ -0,0 +1,244 @@
+import { useMemo } from "react";
```

Comment:
nit: we should hide the "Export" button when it has not been setup yet for the org 

<img width="1264" alt="image" src="https://github.com/user-attachments/assets/294746d3-563c-45d7-bd24-60ec87051b35" />